client/src/lib/session/hooks/useLeaveSession.ts

```diff
@@ -13,14 +13,20 @@ import {
 import useSessions from '../../../lib/sessions/hooks/useSessions';
 import useLiveSessionMetricEvents from './useLiveSessionMetricEvents';
 import useIsSessionHost from './useIsSessionHost';
-import {SessionMode} from '../../../../../shared/src/types/Session';
+import {
+  SessionMode,
+  SessionType,
+} from '../../../../../shared/src/types/Session';
 import useAsyncSessionMetricEvents from './useAsyncSessionMetricEvents';
 
 type ScreenNavigationProps = NativeStackNavigationProp<
   TabNavigatorProps & ModalStackProps
 >;
 
-const useLeaveSession = (sessionMode: SessionMode) => {
+const useLeaveSession = (
+  sessionMode: SessionMode,
+  sessionType: SessionType,
+) => {
```

Comment:
How about sending the full session object here instead? Not a biggie, just feels a bit cleaner so I leave it up to you.

Reply:
Had that thought, so now there are two similar thoughts on that topic 😄  That's enough for me to change it